src/test/java/api/requests/PickSlipsTests.java

```diff
@@ -35,326 +40,356 @@
+    assertEquals(ItemStatus.PAGED.getValue(), itemContext.getString("status"));
+    assertEquals(item.getPrimaryContributorName(), itemContext.getString("primaryContributor"));
+    assertEquals(contributorNames, itemContext.getString("allContributors"));
+    assertEquals(item.getEnumeration(), itemContext.getString("enumeration"));
+    assertEquals(item.getVolume(), itemContext.getString("volume"));
+    assertEquals(item.getChronology(), itemContext.getString("chronology"));
+    assertEquals(yearCaptionsToken, itemContext.getString("yearCaption"));
+    assertEquals(materialTypeName, itemContext.getString("materialType"));
+    assertEquals(loanTypeName, itemContext.getString("loanType"));
+    assertEquals(copyNumber, itemContext.getString("copy"));
+    assertEquals(item.getNumberOfPieces(), itemContext.getString("numberOfPieces"));
+    assertEquals(item.getDescriptionOfPieces(), itemContext.getString("descriptionOfPieces"));
+    assertEquals(actualCheckinDateTime.toString(), itemContext.getString("lastCheckedInDateTime"));
+    assertEquals(location.getName(), itemContext.getString("effectiveLocationSpecific"));
+
+    CallNumberComponents callNumberComponents = item.getCallNumberComponents();
+    assertEquals(callNumberComponents.getCallNumber(), itemContext.getString("callNumber"));
+    assertEquals(callNumberComponents.getPrefix(), itemContext.getString("callNumberPrefix"));
+    assertEquals(callNumberComponents.getSuffix(), itemContext.getString("callNumberSuffix"));
+
+    User requester = new User(requesterResource.getJson());
+    JsonObject requesterContext = pickSlip.getJsonObject("requester");
+
+    assertThat(requesterContext.getString("firstName"), is(requester.getFirstName()));
+    assertThat(requesterContext.getString("lastName"), is(requester.getLastName()));
+    assertThat(requesterContext.getString("middleName"), is(requester.getMiddleName()));
+    assertThat(requesterContext.getString("barcode"), is(requester.getBarcode()));
+    assertThat(requesterContext.getString("addressLine1"), is(address.getAddressLineOne()));
+    assertThat(requesterContext.getString("addressLine2"), is(address.getAddressLineTwo()));
+    assertThat(requesterContext.getString("city"), is(address.getCity()));
+    assertThat(requesterContext.getString("region"), is(address.getRegion()));
+    assertThat(requesterContext.getString("postalCode"), is(address.getPostalCode()));
+    assertThat(requesterContext.getString("countryId"), is(address.getCountryId()));
+
+    JsonObject requestContext = pickSlip.getJsonObject("request");
+
+    assertThat(requestContext.getString("deliveryAddressType"), is(addressTypeResource.getJson().getString("addressType")));
+    assertThat(requestContext.getString("requestExpirationDate"), is(requestExpiration.toDateTimeAtStartOfDay().toString()));
+    assertThat(requestContext.getString("holdShelfExpirationDate"), is(holdShelfExpiration.toDateTimeAtStartOfDay().toString()));
```

Comment:
Please can these long line be split so that they fit within FOLIO's line length limit.

Reply:
the longest line here is 130 characters. According to [this page](https://wiki.folio.org/display/FOLIJET/Java+code+formatter+for+IDEs), the max line length is 132 characters. Or are you referring to some other project-wide standard I'm not aware of?

Reply:
> the longest line here is 130 characters. According to [this page](https://wiki.folio.org/display/FOLIJET/Java+code+formatter+for+IDEs), the max line length is 132 characters.

I don't know where that documentation came from.

> Or are you referring to some other project-wide standard I'm not aware of?

The [official documentation](https://dev.folio.org/guidelines/contributing/#coding-style) references a coding standard that states 80 characters (which is what the .editorconfig file defines)

Reply:
I tend to think of between 80 and 100 as acceptable (especially as the intent is to move to the Google Java coding style in the future)

Reply:
Fixed. Is there a code style config for IDEs available somewhere that I can grab and import?

Reply:
Not for line lengths, IntelliJ has a `.editorconfig` plugin that does this. I think Eclipse probably has similar

Reply:
Yes, I just learned about this plugin yesterday.